qiskit/circuit/library/standard_gates/x.py

```diff
@@ -584,8 +584,8 @@ def __init__(
 
         Args:
             label: An optional label for the gate [Default: ``None``]
-            ctrl_state: control state expressed as integer,
-                string (e.g.``'110'``), or ``None``. If ``None``, use all 1s.
+            ctrl_state: control state expressed as integer, string (e.g.``"110"``), or ``None``.
+            If ``None``, use all 1s.
```

Comment:
This will have broken the formatting because the continuation line is unindented, and I don't think it'll have actually fixed the problem, which is that there's no space betwen `e.g.` and the opening ` `` `.

Reply:
let's try in c35b9cfb4444053df943cb08ba1472fd1046d136